tests/src/Search/Workflow/CollectionWorkflowTest.php

```diff
@@ -5,8 +5,8 @@
 use eLife\ApiSdk\Model\Collection;
 use eLife\Search\Api\ApiValidator;
 use eLife\Search\Api\Elasticsearch\MappedElasticsearchClient;
+use eLife\Search\Workflow\AbstractWorkflow;
```

Comment:
Should we just rename the `AbstractWorkflow` `Workflow` as we are removing `eLife\Search\Workflow\Workflow`?

Reply:
I think it's better to keep it as `AbstractWorkflow`, because it can be recognized even by looking at the filename, and also we have another `Workflow` class in `eLife\Search\Queue`.